README.md

```diff
@@ -3,7 +3,61 @@
 It's an initial Java project to connect to Blibli Seller API. 
 Please feel free to relay your comments, suggestions or corrections through pull request.
 
-Installation step:
-1. Download/clone the initial project
-2. Look at 'App.java' as a main class. It show you to use the client's code
-3. Or download the final .jar here https://github.com/bliblidotcom/seller-api-client-java/releases/tag/1.0.0-1
+### Setup
+
```

Comment:
Option 2: Import Maven dependency
You can import the final sample code .jar into your project by using maven.
Update your pom.xml like the below setup.